config/settings/local.py

```diff
@@ -45,4 +45,8 @@
     },
 }
 
+LANGUAGES = [("de", "German"), ("en", "English")]
+
+LOCALE_PATHS = ["/locale"]
+
```

Comment:
Warum werden die Zeilen nicht einfach in `common.py` gesetzt? Das betritt auch `config/settings/production.py`. Dadurch gäbe es keine Code Dopplung.

Reply:
Das ist aus der Fehlerbehebung eines anderen Problems entstanded. Selbstverständlich rückt das  alles in `common.py`.